internal/storage/v2/api/tracestore/reader.go

```diff
@@ -117,6 +119,19 @@ func (t *TraceQueryParams) ToSpanStoreQueryParameters() *spanstore.TraceQueryPar
 	}
 }
 
+func (t *TraceQueryParams) ToProtoQueryParameters() *storage.TraceQueryParameters {
```

Comment:
`api/tracestore` should not have package dependencies on v1 spanstore nor the grpc storage. Those respective packages should have the conversion that they need to their own types.